frontend/src/Components/PrivacyNoticeContainer/PrivacyNoticeContainer.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { render } from '@testing-library/preact';
+import { PrivacyNoticeContainer } from './PrivacyNoticeContainer';
+import { fireEvent } from '@testing-library/preact';
+
+describe('The PrivacyNoticeContainer', () => {
+  it('displays the privacy notice when clicked', () => {
+    const { getByText, queryAllByText } = render(<PrivacyNoticeContainer />);
+    expect(queryAllByText('Processing of Personal Data')).toHaveLength(0);
```

Comment:
I am only familiar with `@testing-library/react`. With that library, I would have written:

```typescript
expect(queryByText('Processing of Personal Data')).not.toBeInTheDocument();
```
This doesn't work. Can you explain while the interface is a bit different for `preact`?